netapp/netapp.go

```diff
@@ -81,12 +85,16 @@ func NewClient(endpoint string, version string, options *ClientOptions) *Client
 		options.Timeout = 60 * time.Second
 	}
 
+	tlsConfig, err := NewTLSConfig(options)
+	if err != nil {
+		fmt.Println(err)
+		return nil
```

Comment:
This should probably be the standard Go 'return nil, <error>' idiom vs. using Println, but that would involve changing the return signature and I didn't want this to be a breaking change.

Reply:
Thank you for your consideration. I will raise the version, so let's return an error without worrying about backward compatibility.